bedrock/firefox/tests/test_base.py

```diff
@@ -798,6 +798,24 @@ def test_fx_110_0_0_en(self, render_mock):
         template = render_mock.call_args[0][1]
         assert template == ["firefox/whatsnew/whatsnew-fx110-en.html"]
 
+    @override_settings(DEV=True)
+    def test_fx_110_0_0_en_v1(self, render_mock):
+        """Should use whatsnew-fx110-en-features-v1 template for en-US locale"""
+        req = self.rf.get("/firefox/whatsnew/?v=1")
+        req.locale = "en-US"
+        self.view(req, version="110.0")
+        template = render_mock.call_args[0][1]
+        assert template == ["firefox/whatsnew/whatsnew-fx110-en-features-v1.html"]
+
+    @override_settings(DEV=True)
+    def test_fx_110_0_0_en_v2(self, render_mock):
+        """Should use whatsnew-fx110-en-features-v2 template for en-US locale"""
+        req = self.rf.get("/firefox/whatsnew/?v=2")
+        req.locale = "en-US"
+        self.view(req, version="110.0")
+        template = render_mock.call_args[0][1]
+        assert template == ["firefox/whatsnew/whatsnew-fx110-en-features-v2.html"]
+
```

Comment:
Issue (blocking): These tests fail for me locally (for some reason Circle hasn't run them)

Could we maybe change things round so the query param numbers match the template naming?